lessons/2.mdx

```diff
@@ -65,16 +79,16 @@ comfortable with another package manager, feel free to use it.
 
 <ContentCallout emoji='💡' size='md' variant='info'>
 To install `npm` on your system, please follow the instructions on
-[this link](https://www.notion.so/67b2ab1f76f148f49f547b9156aeaf28).
+[this link](/lessons/install-npm).
 </ContentCallout>
 
 Let’s create and `cd` into our D_D Academy projects folder, and create a
 folder for our NFT project:
 
 ```bash
 # create a folder for all our schoolofcode projects
```

Comment:
Comment refers t old name

Reply:
Thanks for reviewing @PBillingsby 😊
Do you mean old name as in notion page or old name as in old routes?
Because all routes will have to be reviewed after the dynamic routes PR i think

Reply:
Dismiss my old comment!
I'll update this as soon as I can